### ISSUE TITLE
Accessibility Bug: When the component is resized to 200%, the ‘Breadcrumb level 1’ is not visible fully.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3113]
Fundamental Core component: "Breadcrumb"
Description: When the application is resized to 200%, the ‘Breadcrumb level 1’ is not visible fully.

Expected: All the UI elements should be clearly visible in 200% as in 100% without any loss of content or functionality.

Attached the screenshot:
100% zoom
![image](https://user-images.githubusercontent.com/32538291/91551991-b1e95f00-e948-11ea-90bd-0fddcde8c587.png)

200%
![image](https://user-images.githubusercontent.com/32538291/91552030-bf064e00-e948-11ea-87e8-ee3be4178ed9.png)

**Comment by salarenko** [comment]
Breadcrumb collapsing based on the screen-width is a proper behavior according to FIori3 standard.
